### PR TITLE
new pull request templates

### PR DESCRIPTION
**Pull request by LucasHT22**
needs polish but here is

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/games.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---
+name: Games
+about: Submiting your game!
```
Comment (grymmy) [inline .github/PULL_REQUEST_TEMPLATE/games.md:3]: nit: typo

**Review by grymmy — APPROVED** [review]

**Review by grymmy — APPROVED** [review]
lgtm

**Comment by grymmy** [comment]
@JosiasAurel plz look at these and give feedback if desired - are we asking all that we should for all of these?

**Comment by JosiasAurel** [comment]
I think it's worth asking if the PR author has tested their changes to make sure it doesn't break anything else in the project.

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
we should ask if the author has tested their changes and asserted that it doesn't break anything else

**Comment by LucasHT22** [comment]
Sorry for the delay! Done.

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Comment by LucasHT22** [comment]
Was reading about it, and looks like this won't work, also the possibly solution just complicate more. Closing this for now, sorry to make you review for nothing.

Ref: https://stackoverflow.com/questions/73771068/multiple-templates-for-pull-requests-on-github
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request

**Pull request closed without merging**